app/page.tsx

```diff
@@ -43,6 +43,7 @@ export default function Airdrop() {
 
   return (
     <div className={`w-full font-mona bg-secondary ${currentComponent === "landing" ? "h-screen md:min-h-[1500px]" : "min-h-screen bg-radial-gradient-green"}`}>
+      {/*
```

Comment:
@creatix remove the code if it's not needed, instead of commenting it. Just mentioned in the commit message about the reason of replacing it.